### ISSUE TITLE
RPCError: _flutter.listViews: (-32000) Service connection disposed

### ISSUE THREAD

**Issue by rubenlop88** [flutter/flutter#154490]
### Steps to reproduce

flutter run -v

### Expected results

The app should run

### Actual results

RPCError: _flutter.listViews: (-32000) Service connection disposed

```
#0      VmService._call (package:vm_service/src/vm_service.dart:1769:7)
#1      VmService.callServiceExtension (package:vm_service/src/vm_service.dart:1721:14)
#2      VmService.callMethod (package:vm_service/src/vm_service.dart:1712:12)
#3      FlutterVmService.callMethodWrapper (package:flutter_tools/src/vmservice.dart:501:28)
#4      FlutterVmService.getFlutterViews (package:flutter_tools/src/vmservice.dart:913:51)
#5      HotRunner.attach (package:flutter_tools/src/run_hot.dart:304:63)
<asynchronous suspension>
#6      RunCommand.runCommand (package:flutter_tools/src/commands/run.dart:867:27)
<asynchronous suspension>
#7      FlutterCommand.run.<anonymous closure> (package:flutter_tools/src/runner/flutter_command.dart:1408:27)
<asynchronous suspension>
#8      AppContext.run.<anonymous closure> (package:flutter_tools/src/base/context.dart:153:19)
<asynchronous suspension>
#9      CommandRunner.runCommand (package:args/command_runner.dart:212:13)
<asynchronous suspension>
#10     FlutterCommandRunner.runCommand.<anonymous closure> (package:flutter_tools/src/runner/flutter_command_runner.dart:420:9)
<asynchronous suspension>
#11     AppContext.run.<anonymous closure> (package:flutter_tools/src/base/context.dart:153:19)
<asynchronous suspension>
#12     FlutterCommandRunner.runCommand (package:flutter_tools/src/runner/flutter_command_runner.dart:364:5)
<asynchronous suspension>
#13     run.<anonymous closure>.<anonymous closure> (package:flutter_tools/runner.dart:130:9)
<asynchronous suspension>
#14     AppContext.run.<anonymous closure> (package:flutter_tools/src/base/context.dart:153:19)
<asynchronous suspension>
#15     main (package:flutter_tools/executable.dart:93:3)
<asynchronous suspension>
```

### Code sample

<details open><summary>Code sample</summary>

```dart
[Paste your code here]
```

</details>

### Screenshots or Video

<details open>
<summary>Screenshots / Video demonstration</summary>

[Upload media here]

</details>

### Logs

<details open><summary>Logs</summary>

```console
[Paste your logs here]
```

</details>

### Flutter Doctor output

<details open><summary>Doctor output</summary>

```
[32m[✓][39m Flutter (Channel stable, 3.24.1, on macOS 14.4.1 23E224 darwin-arm64, locale es-419)
    [32m•[39m Flutter version 3.24.1 on channel stable at /Users/rubenlopez/fvm/versions/stable
    [32m•[39m Upstream repository https://github.com/flutter/flutter.git
    [32m•[39m Framework revision 5874a72aa4 (12 days ago), 2024-08-20 16:46:00 -0500
    [32m•[39m Engine revision c9b9d5780d
    [32m•[39m Dart version 3.5.1
    [32m•[39m DevTools version 2.37.2

[32m[✓][39m Android toolchain - develop for Android devices (Android SDK version 35.0.0-rc3)
    [32m•[39m Android SDK at /Users/rubenlopez/Library/Android/sdk
    [32m•[39m Platform android-35, build-tools 35.0.0-rc3
    [32m•[39m Java binary at: /Users/rubenlopez/Applications/Android Studio.app/Contents/jbr/Contents/Home/bin/java
    [32m•[39m Java version OpenJDK Runtime Environment (build 17.0.11+0-17.0.11b1207.24-11852314)
    [32m•[39m All Android licenses accepted.

[32m[✓][39m Xcode - develop for iOS and macOS (Xcode 15.4)
    [32m•[39m Xcode at /Applications/Xcode.app/Contents/Developer
    [32m•[39m Build 15F31d
    [32m•[39m CocoaPods version 1.15.2

[32m[✓][39m Chrome - develop for the web
    [32m•[39m Chrome at /Applications/Google Chrome.app/Contents/MacOS/Google Chrome

[32m[✓][39m Android Studio (version 2024.1)
    [32m•[39m Android Studio at /Users/rubenlopez/Applications/Android Studio.app/Contents
    [32m•[39m Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    [32m•[39m Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart
    [32m•[39m Java version OpenJDK Runtime Environment (build 17.0.11+0-17.0.11b1207.24-11852314)

[32m[✓][39m VS Code (version 1.92.2)
    [32m•[39m VS Code at /Applications/Visual Studio Code.app/Contents
    [32m•[39m Flutter extension version 3.94.0

[32m[✓][39m Connected device (5 available)
    [32m•[39m sdk gphone64 arm64 (mobile)     • emulator-5554             • android-arm64  • Android 14 (API 34) (emulator)
    [32m•[39m iPhone de Ruben (mobile)        • 00008110-000E6D5E36F8401E • ios            • iOS 17.6.1 21G93
    [32m•[39m macOS (desktop)                 • macos                     • darwin-arm64   • macOS 14.4.1 23E224 darwin-arm64
    [32m•[39m Mac Designed for iPad (desktop) • mac-designed-for-ipad     • darwin         • macOS 14.4.1 23E224 darwin-arm64
    [32m•[39m Chrome (web)                    • chrome                    • web-javascript • Google Chrome 128.0.6613.86

[32m[✓][39m Network resources
    [32m•[39m All expected network resources are available.

[32m•[39m No issues found!
```

</details>

**Comment by rubenlop88** [comment]
Thanks @danagbemava-nc

**Comment by Mr1ss** [comment]
I switched to Flutter master,the issue still exists。
Flutter crash report.
Please report a bug at [Issues · flutter/flutter](https://github.com/flutter/flutter/issues).

## command

flutter run -v

## exception

RPCError: getVM: (-32000) Service connection disposed

```
#0      VmService._call (package:vm_service/src/vm_service.dart:1769:7)
#1      VmService.getVM (package:vm_service/src/vm_service.dart:1245:25)
#2      FlutterDevice.initLogReader (package:flutter_tools/src/resident_runner.dart:420:55)
#3      HotRunner.attach (package:flutter_tools/src/run_hot.dart:264:21)
<asynchronous suspension>
#4      RunCommand.runCommand (package:flutter_tools/src/commands/run.dart:850:27)
<asynchronous suspension>
#5      FlutterCommand.run.<anonymous closure> (package:flutter_tools/src/runner/flutter_command.dart:1450:27)
<asynchronous suspension>
#6      AppContext.run.<anonymous closure> (package:flutter_tools/src/base/context.dart:153:19)
<asynchronous suspension>
#7      CommandRunner.runCommand (package:args/command_runner.dart:212:13)
<asynchronous suspension>
#8      FlutterCommandRunner.runCommand.<anonymous closure> (package:flutter_tools/src/runner/flutter_command_runner.dart:421:9)
<asynchronous suspension>
#9      AppContext.run.<anonymous closure> (package:flutter_tools/src/base/context.dart:153:19)
<asynchronous suspension>
#10     FlutterCommandRunner.runCommand (package:flutter_tools/src/runner/flutter_command_runner.dart:364:5)
<asynchronous suspension>
#11     run.<anonymous closure>.<anonymous closure> (package:flutter_tools/runner.dart:130:9)
<asynchronous suspension>
#12     AppContext.run.<anonymous closure> (package:flutter_tools/src/base/context.dart:153:19)
<asynchronous suspension>
#13     main (package:flutter_tools/executable.dart:94:3)
<asynchronous suspension>
```

## flutter doctor

```
[32m[✓][39m Flutter (Channel master, 3.25.0-1.0.pre.219, on macOS 14.6.1 23G93 darwin-arm64, locale ja-CN)
    [32m•[39m Flutter version 3.25.0-1.0.pre.219 on channel master at /Users/guan/development/flutter
    [32m•[39m Upstream repository [GitHub - flutter/flutter: Flutter makes it easy and fast to build beautiful apps for mobile and beyo](https://github.com/flutter/flutter.git)
    [32m•[39m Framework revision 6fe09872b1 (3 days ago), 2024-08-30 19:53:11 -0400
    [32m•[39m Engine revision 2d56e44888
    [32m•[39m Dart version 3.6.0 (build 3.6.0-198.0.dev)
    [32m•[39m DevTools version 2.39.0

[32m[✓][39m Android toolchain - develop for Android devices (Android SDK version 34.0.0)
    [32m•[39m Android SDK at /Users/guan/Library/Android/sdk
    [32m•[39m Platform android-34, build-tools 34.0.0
    [32m•[39m ANDROID_HOME = /Users/guan/Library/Android/sdk
    [32m•[39m Java binary at: /Applications/Android Studio.app/Contents/jbr/Contents/Home/bin/java
    [32m•[39m Java version OpenJDK Runtime Environment (build 17.0.7+0-17.0.7b1000.6-10550314)
    [32m•[39m All Android licenses accepted.

[32m[✓][39m Xcode - develop for iOS and macOS (Xcode 15.2)
    [32m•[39m Xcode at /Applications/Xcode.app/Contents/Developer
    [32m•[39m Build 15C500b
    [32m•[39m CocoaPods version 1.15.2

[32m[✓][39m Chrome - develop for the web
    [32m•[39m Chrome at /Applications/Google Chrome.app/Contents/MacOS/Google Chrome

[32m[✓][39m Android Studio (version 2023.1)
    [32m•[39m Android Studio at /Applications/Android Studio.app/Contents
    [32m•[39m Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    [32m•[39m Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart
    [32m•[39m Java version OpenJDK Runtime Environment (build 17.0.7+0-17.0.7b1000.6-10550314)

[32m[✓][39m VS Code (version 1.92.2)
    [32m•[39m VS Code at /Applications/Visual Studio Code.app/Contents
    [32m•[39m Flutter extension version 3.94.0

[32m[✓][39m Connected device (4 available)
    [32m•[39m iPhone 14 Pro (mobile)          • F4416661-5F6D-4C8B-A67E-E50F7173CBF1 • ios            • com.apple.CoreSimulator.SimRuntime.iOS-17-2 (simulator)
    [32m•[39m macOS (desktop)                 • macos                                • darwin-arm64   • macOS 14.6.1 23G93 darwin-arm64
    [32m•[39m Mac Designed for iPad (desktop) • mac-designed-for-ipad                • darwin         • macOS 14.6.1 23G93 darwin-arm64
    [32m•[39m Chrome (web)                    • chrome                               • web-javascript • Google Chrome 128.0.6613.114
    [33m![39m Error: Browsing on the local area network for iPhone14pro_17. Ensure the device is unlocked and attached with a cable or associated with the same
      local area network as this Mac.
      The device must be opted into Developer Mode to connect wirelessly. (code -27)

[33m[!][39m Network resources
    [31m✗[39m A network error occurred while checking "https://maven.google.com/": Operation timed out
    [31m✗[39m A network error occurred while checking "[GitHub: Let’s build from here](https://github.com/)": Operation timed out

[33m![39m Doctor found issues in 1 category.
```

**Comment by danagbemava-nc** [comment]
Reopening this based on https://github.com/flutter/flutter/issues/154490#issuecomment-2325779601

cc @andrewkolos, seems like this might still occur on the master channel

**Comment by Mr1ss** [comment]
sorry my mistake, it happend when the app crash.

**Comment by Mr1ss** [comment]
sorry my mistake, it happend when the app crash.

**Comment by Mr1ss** [comment]
@danagbemava-nc

**Comment by Mr1ss** [comment]
Flutter crash report.
Please report a bug at [Issues · flutter/flutter](https://github.com/flutter/flutter/issues).

## command

flutter run -v

## exception

RPCError: getVM: (-32000) Service connection disposed

```
#0      VmService._call (package:vm_service/src/vm_service.dart:1769:7)
#1      VmService.getVM (package:vm_service/src/vm_service.dart:1245:25)
#2      FlutterDevice.initLogReader (package:flutter_tools/src/resident_runner.dart:420:55)
#3      HotRunner.attach (package:flutter_tools/src/run_hot.dart:264:21)
<asynchronous suspension>
#4      RunCommand.runCommand (package:flutter_tools/src/commands/run.dart:850:27)
<asynchronous suspension>
#5      FlutterCommand.run.<anonymous closure> (package:flutter_tools/src/runner/flutter_command.dart:1450:27)
<asynchronous suspension>
#6      AppContext.run.<anonymous closure> (package:flutter_tools/src/base/context.dart:153:19)
<asynchronous suspension>
#7      CommandRunner.runCommand (package:args/command_runner.dart:212:13)
<asynchronous suspension>
#8      FlutterCommandRunner.runCommand.<anonymous closure> (package:flutter_tools/src/runner/flutter_command_runner.dart:421:9)
<asynchronous suspension>
#9      AppContext.run.<anonymous closure> (package:flutter_tools/src/base/context.dart:153:19)
<asynchronous suspension>
#10     FlutterCommandRunner.runCommand (package:flutter_tools/src/runner/flutter_command_runner.dart:364:5)
<asynchronous suspension>
#11     run.<anonymous closure>.<anonymous closure> (package:flutter_tools/runner.dart:130:9)
<asynchronous suspension>
#12     AppContext.run.<anonymous closure> (package:flutter_tools/src/base/context.dart:153:19)
<asynchronous suspension>
#13     main (package:flutter_tools/executable.dart:94:3)
<asynchronous suspension>
```

## flutter doctor

```
[32m[✓][39m Flutter (Channel master, 3.25.0-1.0.pre.219, on macOS 14.6.1 23G93 darwin-arm64, locale ja-CN)
    [32m•[39m Flutter version 3.25.0-1.0.pre.219 on channel master at /Users/guan/development/flutter
    [32m•[39m Upstream repository [GitHub - flutter/flutter: Flutter makes it easy and fast to build beautiful apps for mobile and beyo](https://github.com/flutter/flutter.git)
    [32m•[39m Framework revision 6fe09872b1 (3 days ago), 2024-08-30 19:53:11 -0400
    [32m•[39m Engine revision 2d56e44888
    [32m•[39m Dart version 3.6.0 (build 3.6.0-198.0.dev)
    [32m•[39m DevTools version 2.39.0

[32m[✓][39m Android toolchain - develop for Android devices (Android SDK version 34.0.0)
    [32m•[39m Android SDK at /Users/guan/Library/Android/sdk
    [32m•[39m Platform android-34, build-tools 34.0.0
    [32m•[39m ANDROID_HOME = /Users/guan/Library/Android/sdk
    [32m•[39m Java binary at: /Applications/Android Studio.app/Contents/jbr/Contents/Home/bin/java
    [32m•[39m Java version OpenJDK Runtime Environment (build 17.0.7+0-17.0.7b1000.6-10550314)
    [32m•[39m All Android licenses accepted.

[32m[✓][39m Xcode - develop for iOS and macOS (Xcode 15.2)
    [32m•[39m Xcode at /Applications/Xcode.app/Contents/Developer
    [32m•[39m Build 15C500b
    [32m•[39m CocoaPods version 1.15.2

[32m[✓][39m Chrome - develop for the web
    [32m•[39m Chrome at /Applications/Google Chrome.app/Contents/MacOS/Google Chrome

[32m[✓][39m Android Studio (version 2023.1)
    [32m•[39m Android Studio at /Applications/Android Studio.app/Contents
    [32m•[39m Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    [32m•[39m Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart
    [32m•[39m Java version OpenJDK Runtime Environment (build 17.0.7+0-17.0.7b1000.6-10550314)

[32m[✓][39m VS Code (version 1.92.2)
    [32m•[39m VS Code at /Applications/Visual Studio Code.app/Contents
    [32m•[39m Flutter extension version 3.94.0

[32m[✓][39m Connected device (4 available)
    [32m•[39m iPhone 14 Pro (mobile)          • F4416661-5F6D-4C8B-A67E-E50F7173CBF1 • ios            • com.apple.CoreSimulator.SimRuntime.iOS-17-2 (simulator)
    [32m•[39m macOS (desktop)                 • macos                                • darwin-arm64   • macOS 14.6.1 23G93 darwin-arm64
    [32m•[39m Mac Designed for iPad (desktop) • mac-designed-for-ipad                • darwin         • macOS 14.6.1 23G93 darwin-arm64
    [32m•[39m Chrome (web)                    • chrome                               • web-javascript • Google Chrome 128.0.6613.114
    [33m![39m Error: Browsing on the local area network for iPhone14pro_17. Ensure the device is unlocked and attached with a cable or associated with the same
      local area network as this Mac.
      The device must be opted into Developer Mode to connect wirelessly. (code -27)

[33m[!][39m Network resources
    [31m✗[39m A network error occurred while checking "https://maven.google.com/": Operation timed out
    [31m✗[39m A network error occurred while checking "[GitHub: Let’s build from here](https://github.com/)": Operation timed out

[33m![39m Doctor found issues in 1 category.
```

**Comment by danagbemava-nc** [comment]
Hi @Mr1ss, please share the crash log. According to https://github.com/flutter/flutter/issues/153471#issue-2466945913, that is typically when the crash happens

**Comment by Mr1ss** [comment]
@danagbemava-nc ok

**Comment by soft7store** [comment]
I faced same issue, how can I fix it? It crashed only in ios.

**Comment by soft7store** [comment]
RPCError: _flutter.listViews: (-32000) Service connection disposed

```
#0      VmService._call (package:vm_service/src/vm_service.dart:1769:7)
#1      VmService.callServiceExtension (package:vm_service/src/vm_service.dart:1721:14)
#2      VmService.callMethod (package:vm_service/src/vm_service.dart:1712:12)
#3      FlutterVmService.callMethodWrapper (package:flutter_tools/src/vmservice.dart:501:28)
#4      FlutterVmService.getFlutterViews (package:flutter_tools/src/vmservice.dart:913:51)
#5      HotRunner.attach (package:flutter_tools/src/run_hot.dart:304:63)
<asynchronous suspension>
#6      RunCommand.runCommand (package:flutter_tools/src/commands/run.dart:867:27)
<asynchronous suspension>
#7      FlutterCommand.run.<anonymous closure> (package:flutter_tools/src/runner/flutter_command.dart:1408:27)
<asynchronous suspension>
#8      AppContext.run.<anonymous closure> (package:flutter_tools/src/base/context.dart:153:19)
<asynchronous suspension>
#9      CommandRunner.runCommand (package:args/command_runner.dart:212:13)
<asynchronous suspension>
#10     FlutterCommandRunner.runCommand.<anonymous closure> (package:flutter_tools/src/runner/flutter_command_runner.dart:420:9)
<asynchronous suspension>
#11     AppContext.run.<anonymous closure> (package:flutter_tools/src/base/context.dart:153:19)
<asynchronous suspension>
#12     FlutterCommandRunner.runCommand (package:flutter_tools/src/runner/flutter_command_runner.dart:364:5)
<asynchronous suspension>
#13     run.<anonymous closure>.<anonymous closure> (package:flutter_tools/runner.dart:130:9)
<asynchronous suspension>
#14     AppContext.run.<anonymous closure> (package:flutter_tools/src/base/context.dart:153:19)
<asynchronous suspension>
#15     main (package:flutter_tools/executable.dart:93:3)
<asynchronous suspension>

**Comment by andrewkolos** [comment]
Assigning to myself for further triage.

**Comment by soft7store** [comment]
https://github.com/flutter/flutter/issues/new?title=%5Btool_crash%5D+RPCError&body=%23%23+Command%0A%60%60%60sh%0Aflutter+run%0A%60%60%60%0A%0A%23%23+Steps+to+Reproduce%0A1.+...%0A2.+...%0A3.+...%0A%0A%23%23+Logs%0ARPCError%0A%60%60%60console%0A%230++++++VmService._call+%28package%3Avm_service%2Fsrc%2Fvm_service.dart%3A1769%3A7%29%0A%231++++++VmService.callServiceExtension+%28package%3Avm_service%2Fsrc%2Fvm_service.dart%3A1721%3A14%29%0A%232++++++VmService.callMethod+%28package%3Avm_service%2Fsrc%2Fvm_service.dart%3A1712%3A12%29%0A%233++++++FlutterVmService.callMethodWrapper+%28package%3Aflutter_tools%2Fsrc%2Fvmservice.dart%3A501%3A28%29%0A%234++++++FlutterVmService.getFlutterViews+%28package%3Aflutter_tools%2Fsrc%2Fvmservice.dart%3A913%3A51%29%0A%235++++++HotRunner.attach+%28package%3Aflutter_tools%2Fsrc%2Frun_hot.dart%3A304%3A63%29%0A%3Casynchronous+suspension%3E%0A%236++++++RunCommand.runCommand+%28package%3Aflutter_tools%2Fsrc%2Fcommands%2Frun.dart%3A867%3A27%29%0A%3Casynchronous+suspension%3E%0A%237++++++FlutterCommand.run.%3Canonymous+closure%3E+%28package%3Aflutter_tools%2Fsrc%2Frunner%2Fflutter_command.dart%3A1408%3A27%29%0A%3Casynchronous+suspension%3E%0A%238++++++AppContext.run.%3Canonymous+closure%3E+%28package%3Aflutter_tools%2Fsrc%2Fbase%2Fcontext.dart%3A153%3A19%29%0A%3Casynchronous+suspension%3E%0A%239++++++CommandRunner.runCommand+%28package%3Aargs%2Fcommand_runner.dart%3A212%3A13%29%0A%3Casynchronous+suspension%3E%0A%2310+++++FlutterCommandRunner.runCommand.%3Canonymous+closure%3E+%28package%3Aflutter_tools%2Fsrc%2Frunner%2Fflutter_command_runner.dart%3A420%3A9%29%0A%3Casynchronous+suspension%3E%0A%2311+++++AppContext.run.%3Canonymous+closure%3E+%28package%3Aflutter_tools%2Fsrc%2Fbase%2Fcontext.dart%3A153%3A19%29%0A%3Casynchronous+suspension%3E%0A%2312+++++FlutterCommandRunner.runCommand+%28package%3Aflutter_tools%2Fsrc%2Frunner%2Fflutter_command_runner.dart%3A364%3A5%29%0A%3Casynchronous+suspension%3E%0A%2313+++++run.%3Canonymous+closure%3E.%3Canonymous+closure%3E+%28package%3Aflutter_tools%2Frunner.dart%3A130%3A9%29%0A%3Casynchronous+suspension%3E%0A%2314+++++AppContext.run.%3Canonymous+closure%3E+%28package%3Aflutter_tools%2Fsrc%2Fbase%2Fcontext.dart%3A153%3A19%29%0A%3Casynchronous+suspension%3E%0A%60%60%60%0A%60%60%60console%0A%1B%5B32m%5B%E2%9C%93%5D%1B%5B39m+Flutter+%28Channel+stable%2C+3.24.1%2C+on+macOS+14.0+23A5257q+darwin-x64%2C+locale+en-US%29%0A++++%1B%5B32m%E2%80%A2%1B%5B39m+Flutter+version+3.24.1+on+channel+stable+at+%2FUsers%2Fman%2FDocuments%2Fflutter_macos_3.24.1-stable%0A++++%1B%5B32m%E2%80%A2%1B%5B39m+Upstream+repository+https%3A%2F%2Fgithub.com%2Fflutter%2Fflutter.git%0A++++%1B%5B32m%E2%80%A2%1B%5B39m+Framework+revision+5874a72aa4+%282+weeks+ago%29%2C+2024-08-20+16%3A46%3A00+-0500%0A++++%1B%5B32m%E2%80%A2%1B%5B39m+Engine+revision+c9b9d5780d%0A++++%1B%5B32m%E2%80%A2%1B%5B39m+Dart+version+3.5.1%0A++++%1B%5B32m%E2%80%A2%1B%5B39m+DevTools+version+2.37.2%0A%0A%1B%5B31m%5B%E2%9C%97%5D%1B%5B39m+Android+toolchain+-+develop+for+Android+devices%0A++++%1B%5B31m%E2%9C%97%1B%5B39m+Unable+to+locate+Android+SDK.%0A++++++Install+Android+Studio+from%3A+https%3A%2F%2Fdeveloper.android.com%2Fstudio%2Findex.html%0A++++++On+first+launch+it+will+assist+you+in+installing+the+Android+SDK+components.%0A++++++%28or+visit+https%3A%2F%2Fflutter.dev%2Fto%2Fmacos-android-setup+for+detailed+instructions%29.%0A++++++If+the+Android+SDK+has+been+installed+to+a+custom+location%2C+please+use%0A++++++%60flutter+config+--android-sdk%60+to+update+to+that+location.%0A%0A%0A%1B%5B32m%5B%E2%9C%93%5D%1B%5B39m+Xcode+-+develop+for+iOS+and+macOS+%28Xcode+15.4%29%0A++++%1B%5B32m%E2%80%A2%1B%5B39m+Xcode+at+%2FApplications%2FXcode.app%2FContents%2FDeveloper%0A++++%1B%5B32m%E2%80%A2%1B%5B39m+Build+15F31d%0A++++%1B%5B32m%E2%80%A2%1B%5B39m+CocoaPods+version+1.15.2%0A%0A%1B%5B32m%5B%E2%9C%93%5D%1B%5B39m+Chrome+-+develop+for+the+web%0A++++%1B%5B32m%E2%80%A2%1B%5B39m+Chrome+at+%2FApplications%2FGoogle+Chrome.app%2FContents%2FMacOS%2FGoogle+Chrome%0A%0A%1B%5B33m%5B%21%5D%1B%5B39m+Android+Studio+%28not+installed%29%0A++++%1B%5B32m%E2%80%A2%1B%5B39m+Android+Studio+not+found%3B+download+from+https%3A%2F%2Fdeveloper.android.com%2Fstudio%2Findex.html%0A++++++%28or+visit+https%3A%2F%2Fflutter.dev%2Fto%2Fmacos-android-setup+for+detailed+instructions%29.%0A%0A%1B%5B32m%5B%E2%9C%93%5D%1B%5B39m+VS+Code+%28version+1.92.2%29%0A++++%1B%5B32m%E2%80%A2%1B%5B39m+VS+Code+at+%2FApplications%2FVisual+Studio+Code.app%2FContents%0A++++%1B%5B32m%E2%80%A2%1B%5B39m+Flutter+extension+version+3.96.0%0A%0A%1B%5B32m%5B%E2%9C%93%5D%1B%5B39m+Connected+device+%283+available%29%0A++++%1B%5B32m%E2%80%A2%1B%5B39m+iPhone+15+Pro+%28mobile%29+%E2%80%A2+2F4F7873-C66E-46A9-ABAA-DB3EEB5C73B0+%E2%80%A2+ios++++++++++++%E2%80%A2+com.apple.CoreSimulator.SimRuntime.iOS-17-5+%28simulator%29%0A++++%1B%5B32m%E2%80%A2%1B%5B39m+macOS+%28desktop%29++++++++%E2%80%A2+macos++++++++++++++++++++++++++++++++%E2%80%A2+darwin-x64+++++%E2%80%A2+macOS+14.0+23A5257q+darwin-x64%0A++++%1B%5B32m%E2%80%A2%1B%5B39m+Chrome+%28web%29+++++++++++%E2%80%A2+chrome+++++++++++++++++++++++++++++++%E2%80%A2+web-javascript+%E2%80%A2+Google+Chrome+128.0.6613.119%0A%0A%1B%5B32m%5B%E2%9C%93%5D%1B%5B39m+Network+resources%0A++++%1B%5B32m%E2%80%A2%1B%5B39m+All+expected+network+resources+are+available.%0A%0A%1B%5B33m%21%1B%5B39m+Doctor+found+issues+in+2+categories.%0A%0A%60%60%60%0A%0A%23%23+Flutter+Application+Metadata%0A%2A%2AType%2A%2A%3A+malformed%0A%2A%2AVersion%2A%2A%3A+0.1.0%0A%2A%2AMaterial%2A%2A%3A+true%0A%2A%2AAndroid+X%2A%2A%3A+false%0A%2A%2AModule%2A%2A%3A+false%0A%2A%2APlugin%2A%2A%3A+false%0A%2A%2AAndroid+package%2A%2A%3A+null%0A%2A%2AiOS+bundle+identifier%2A%2A%3A+null%0A%2A%2ACreation+channel%2A%2A%3A+null%0A%2A%2ACreation+framework+version%2A%2A%3A+null%0A%23%23%23+Plugins%0Aaudioplayers-6.1.0%0Aaudioplayers_android-5.0.0%0Aaudioplayers_darwin-6.0.0%0Aaudioplayers_linux-4.0.0%0Aaudioplayers_web-5.0.1%0Aaudioplayers_windows-4.0.0%0Aconnectivity-3.0.6%0Aconnectivity_for_web-0.4.0%2B1%0Aconnectivity_macos-0.2.1%2B2%0Aconnectivity_plus-6.0.5%0Afile_selector_linux-0.9.2%2B1%0Afile_selector_macos-0.9.4%0Afile_selector_windows-0.9.3%2B2%0Afirebase_core-3.4.0%0Afirebase_core_web-2.17.5%0Afirebase_messaging-15.1.0%0Afirebase_messaging_web-3.8.13%0Aflutter_plugin_android_lifecycle-2.0.22%0Aimage_picker-1.1.2%0Aimage_picker_android-0.8.12%2B13%0Aimage_picker_for_web-3.0.5%0Aimage_picker_ios-0.8.12%0Aimage_picker_linux-0.2.1%2B1%0Aimage_picker_macos-0.2.1%2B1%0Aimage_picker_windows-0.2.1%2B1%0Apath_provider-2.1.4%0Apath_provider_android-2.2.10%0Apath_provider_foundation-2.4.0%0Apath_provider_linux-2.2.1%0Apath_provider_windows-2.3.0%0Ashared_preferences-2.3.2%0Ashared_preferences_android-2.3.2%0Ashared_preferences_foundation-2.5.2%0Ashared_preferences_linux-2.4.1%0Ashared_preferences_web-2.4.2%0Ashared_preferences_windows-2.4.1%0Asqflite-2.3.3%2B1%0Aurl_launcher-6.3.0%0Aurl_launcher_android-6.3.10%0Aurl_launcher_ios-6.3.1%0Aurl_launcher_linux-3.2.0%0Aurl_launcher_macos-3.2.0%0Aurl_launcher_web-2.3.3%0Aurl_launcher_windows-3.1.2%0Awebview_flutter-4.9.0%0Awebview_flutter_android-3.16.7%0Awebview_flutter_wkwebview-3.15.0%0A%0A&labels=tool%2Csevere%3A+crash

**Comment by andrewkolos** [comment]
> https<area>://github.com/flutter/flutter/issues/new?title=%5Btool_crash%5D+RPC...

This occurred on version 3.24.1, where this crash is not fixed.

**Comment by soft7store** [comment]
hmm... then how can I run the app?

**Comment by soft7store** [comment]
On 3.24.2, it was fixed?

**Comment by Mr1ss** [comment]
Any crash logs from xoode？

**Comment by Mr1ss** [comment]
Any crash logs from xoode？

**Comment by simranjeet09** [comment]
[+8992 ms] "flutter run" took 85,642ms.

[ +133 ms] Sending crash report to Google.
[   +8 ms] Service protocol connection closed.
[        ] Lost connection to device.
[ +237 ms] The VM Service for iPhone 14 disappeared while trying to find the ext.flutter.connectedVmServiceUri service extension. Skipping subsequent DevTools setup for
this device.
[ +426 ms] Crash report sent (report ID: 4eb7ddfd5dcf7095)
[        ] Oops; flutter has exited unexpectedly: "getVM: (-32000) Service connection disposed".
[ +109 ms] executing: /usr/bin/plutil -convert xml1 -o - /Applications/Android Studio.app/Contents/Info.plist
[  +10 ms] Exit code 0 from: /usr/bin/plutil -convert xml1 -o - /Applications/Android Studio.app/Contents/Info.plist
[        ] <?xml version="1.0" encoding="UTF-8"?>

**Comment by simranjeet09** [comment]
Issue still exist

**Comment by andrewkolos** [comment]
While some causes of this were fixed in 3.24.1, this crash indeed continues to occur on 3.24.2. I've updated the list of issues on the umbrella issue: https://github.com/flutter/flutter/issues/153471#issue-2466945913. I'm sorry that these crashes continue to occur, and I appreciate folks' patience as I continue to work on getting these fixed.

I'm closing this particular issue since the stack trace shown in the report should no longer appear in 3.24.2 crashes. Other stack traces provided in this thread are now tracked by these new issues. For example, the stack trace provided in https://github.com/flutter/flutter/issues/154490#issuecomment-2328052342 is now tracked by https://github.com/flutter/flutter/issues/154903.
